src/iexamine.cpp

```diff
@@ -805,6 +805,52 @@ void iexamine::cardreader_robofac( player &p, const tripoint &examp )
     }
 }
 
+void iexamine::cardreader_foodplace( player &p, const tripoint &examp )
+{
+    bool open = false;
+    if( ( p.is_wearing( itype_id( "foodperson_mask" ) ) ||
+          p.is_wearing( itype_id( "foodperson_mask_on" ) ) ) &&
+        query_yn( _( "Press mask on the reader?" ) ) ) {
+        p.mod_moves( -100 );
+        for( const tripoint &tmp : g->m.points_in_radius( examp, 3 ) ) {
+            if( g->m.ter( tmp ) == t_door_metal_locked ) {
+                g->m.ter_set( tmp, t_door_metal_c );
+                open = true;
+            }
+        }
+        if( open ) {
+            add_msg( _( "You press your face on the reader." ) );
+            add_msg( m_good, _( "The nearby doors are unlocked." ) );
+            sounds::sound( examp, 6, sounds::sound_t::speech, "\"Hello Foodperson.  Welcome home.\"", true,
+                           "speech", "welcome" );
+        } else {
+            add_msg( _( "The nearby doors are already unlocked." ) );
+            if( query_yn( _( "Lock doors?" ) ) ) {
+                for( const tripoint &tmp : g->m.points_in_radius( examp, 3 ) ) {
+                    if( g->m.ter( tmp ) == t_door_metal_o || g->m.ter( tmp ) == t_door_metal_c ) {
+                        if( p.pos() == tmp ) {
+                            p.add_msg_if_player( m_bad, _( "You are in the way of the door, move before trying again." ) );
+                        } else {
+                            g->m.ter_set( tmp, t_door_metal_locked );
+                        }
+                    }
+                }
+            }
+        }
+    } else if( p.has_amount( itype_id( "foodperson_mask" ), 1 ) ||
+               p.has_amount( itype_id( "foodperson_mask_on" ), 1 ) ) {
+        sounds::sound( examp, 6, sounds::sound_t::speech,
+                       "\"FOODPERSON DETECTED.  Please make yourself presentable.\"", true,
+                       "speech", "welcome" );
+    } else {
+        sounds::sound( examp, 6, sounds::sound_t::speech,
+                       "\"Your face is inadequate.  Please go away.\"", true,
```

Comment:
The description should be translated. Same for the calls below.

Reply:
How do I do that ?

Reply:
```suggestion
                      _( "\"Your face is inadequate.  Please go away.\"" ), true,
```

Reply:
Done https://github.com/CleverRaven/Cataclysm-DDA/pull/31489